core/kernel/dt.c

```diff
@@ -614,12 +629,23 @@ static TEE_Result release_external_dt(void)
 		panic();
 	}
 
-	if (core_mmu_remove_mapping(MEM_AREA_EXT_DT, external_dt.blob,
-				    CFG_DTB_MAX_SIZE))
+	if (IS_ENABLED(CFG_TRANSFER_LIST)) {
+		if (tl) {
```

Comment:
should be `if (IS_ENABLED(CFG_TRANSFER_LIST) && tl)`
so that when `CFG_TRANSFER_LIST=y` but the boot uses external FDT, we unmap from line 642.

Reply:
Done.